ConsumerApi.Sdk/Endpoints/Identities/Types/CreateIdentityResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Backbone.ConsumerApi.Sdk.Endpoints.Identities.Types;
+
+public class CreateIdentityResponse
+{
+    public required string Address { get; set; }
+    public DateTime CreatedAt { get; set; }
```

Comment:
You should make this required. Same for all other properties that are not nullable across all types. That's just a safety net to be notified when the property is not populated by a response. 
Currently it would be set to `0001-01-01T00:00:00`, which may lead to unexpected behavior. Making the property required makes it throw at runtime.

Reply:
👍